examples/multiple_databases.py

```diff
@@ -42,17 +56,27 @@ def get(self):
 class NativeCoroutinesRequestHandler(SessionMixin, RequestHandler):
     async def get(self):
         with self.make_session() as session:
+            session.add(User(username='c'))
+            session.add(Foo(foo='d'))
+            session.commit()
             count = await as_future(session.query(User).count)
 
         self.write('{} users so far!'.format(count))
 
 
 if __name__ == '__main__':
-    parse_command_line()
-
-    assert options.database_url, "Need a database URL"
-
-    db.configure(url=options.database_url)
+    db.configure(
+        url='mysql://t_sa:t_sa@localhost/t_sa',
```

Comment:
Could you modify this to be a `:memory:` url so that someone running this example doesn't need to setup MySQL?

Reply:
A [SQLite](https://docs.sqlalchemy.org/en/13/core/engines.html#sqlite) url would also be fine.